config/routes.rb

```diff
@@ -2,14 +2,13 @@
   root 'restaurants#index'
   devise_for :users
 
-  resources :restaurants
-
   namespace :admin do
     resources :users, only: [:index, :destroy]
     resources :restaurants, only: [:index, :destroy]
+    resources :reviews, only: [:destroy]
```

Comment:
Why is this listed as a namespaced action? I don't see any other stuff relating to admins deleting reviews...